LibraryManagement/src/main/java/com/libraryManagement/entity/Room.java

```diff
@@ -1,5 +1,53 @@
 package com.libraryManagement.entity;
+import java.util.List;
 
-public class Room {
+import com.fasterxml.jackson.annotation.JsonManagedReference;
 
-}
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+@Entity
+public class Room {
+		@Id
+		@GeneratedValue(strategy = GenerationType.IDENTITY)
+		private int roomId;
+		
+		private String roomName;
+		
+		@OneToMany(mappedBy = "room" , cascade = CascadeType.ALL,orphanRemoval = true)
```

Comment:
why this orphan removal is here ?

Reply:
Suppose We deleted Room Table That time their child also Automatically Deleted That's Why I use Orphan true